inspirehep/modules/authors/dojson/fields/updateform.py

```diff
@@ -180,12 +180,12 @@ def institution_history(self, key, value):
             continue
         positions.append({
             "institution": {'name': position["name"]},
-            "status": "current" if position["current"] else "",
+            "current": 'Current' if position["current"] else None,
```

Comment:
The status one made more sense imo, I'd try to keep that one instead if possible.